pkg/skaffold/config/options.go

```diff
@@ -35,6 +35,7 @@ type SkaffoldOptions struct {
 	CacheArtifacts     bool
 	EnableRPC          bool
 	Force              bool
+	ForceDev           bool
```

Comment:
do we still need this after the refactor? I think we should just be able to use `Force`.